subworkflows/local/align_star.nf

```diff
@@ -22,24 +23,53 @@ workflow ALIGN_STAR {
     //
     // Map reads with STAR
     //
-    STAR_ALIGN ( reads, index, gtf, star_ignore_sjdbgtf, seq_platform, seq_center, is_aws_igenome )
-    ch_versions = ch_versions.mix(STAR_ALIGN.out.versions.first())
+    ch_orig_bam       = Channel.empty()
```

Comment:
Out of curiosity, what `orig` stands for, I guess ig=ignenomes, o=output?, r ??

Reply:
orig = original 🤣 It's because STAR can generate loads of different types of BAM files.